server/paths/refer.ts

```diff
@@ -17,6 +17,7 @@ const confirmOasysPath = showReferralPath.path('oasys-confirmed')
 const checkAnswersPath = showReferralPath.path('check-answers')
 const completeReferralPath = showReferralPath.path('complete')
 const submitReferralPath = showReferralPath.path('submit')
+const reasonForReferralPath = showReferralPath.path('reason-for-referral')
```

Comment:
I think this should go after `confirmOasysPath` - currently they're in roughly chronological order